server.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+
+const app = express();
+
+app.use(express.json());
+
+let clubs = [{ id: 0, name: 'Manchester United' }, { id: 1, name: 'Chelsea FC' }];
+
+app.get('/', (req, res) => {
+  res.status(200).json({ message: 'It is alive!' });
+});
```

Comment:
Nice and simple GET endpoint to verify that data was successfully posted and deleted.